tests/typeahead-filter.js

```diff
@@ -1,17 +1,23 @@
 'use strict';
 
-/* global require, document, describe, before, beforeEach, after, afterEach, it */
+/* global require, window, document, describe, before, beforeEach, after, afterEach, it */
+
 var $ = require('jquery');
+var _ = require('underscore');
+
 window.$ = window.jQuery = $;
+_.extend(window, {
```

Comment:
This feels like this should be in a `constants.js` or `globals.js` file. Is there a reason it isn't?